packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -184,21 +184,22 @@ const DialogueRecommendationRow = ({ rowProps, classes, showSuggestedTopics }: D
 
   if (!currentUser) return <></>;
   const preTopicRecommendations: TopicRecommendationWithContents[] | undefined = topicData?.GetTwoUserTopicRecommendations; 
-  const topicRecommendations = preTopicRecommendations?.filter(topic => topic.theirVote !== null);
+  const topicRecommendations = preTopicRecommendations?.filter(topic => ['agree', 'disagree'].includes(topic.theirVote) ); // todo: might want better type checking here in future for values of theirVote
   const numRecommendations = topicRecommendations?.length ?? 0;
   const numShown = isExpanded ? numRecommendations : 1
   const numHidden = Math.max(0, numRecommendations - numShown);
 
   const renderExpandCollapseText = () => {
+    if (!topicRecommendations || topicRecommendations.length === 0) return '';
     if (isExpanded) {
       return '▲ hide';
     }
   
-    if (numHidden > 0) {
+    if (numHidden > 0 || topicRecommendations[0].comment.contents.plaintextMainText.length > 84) { // hack... make better in future!
```

Comment:
comment is not super descriptive re: what exactly the hack is or what should be changed in the future

Reply:
Really it should be the case that if we only have one topic, but an ellipsis, there should be an expand button. But since I can't check for ellipsis, I am instead checking for string length... which it now occurs to me is not robust on mobile, not sure how to handle